### ISSUE TITLE
Error creating load balancer in 0.10.2

### ISSUE THREAD

**Issue by janczarn** [terraform-providers/terraform-provider-openstack#77]
The Terraform config linked below works in Terraform 0.9.11, but gives me this error in 0.10.2:

```
* openstack_lb_pool_v2.pool: No Load Balancer on pool 10ae5f3f-2c30-4978-abec-5f5c8e311d18
```

The pool is created, but the deployment goes no further.

### Terraform Version
0.10.2

The deployment completes with no errors in 0.9.11

### Affected Resource(s)
It fails on openstack_lb_pool_v2.pool.

### Terraform Configuration Files
[deploy.tf](https://gist.github.com/janczarn/607cc8c9525f9da87eb9d6c13064e388)

### Panic Output
```
Error applying plan:

1 error(s) occurred:

* openstack_lb_pool_v2.pool: 1 error(s) occurred:

* openstack_lb_pool_v2.pool: No Load Balancer on pool 10ae5f3f-2c30-4978-abec-5f5c8e311d18
```

### Expected Behavior
The deployment should have completed with all load balancer resources.

### Actual Behavior
The deployment failed after creating the load balancer pool.

**Comment by jtopjian** [comment]
@janczarn Thanks for reporting this. There was a good amount of work done to the LBaaSv2 in the last release, so it's possible there might be a bug.

Can you try using `loadbalancer_id` instead of `listener_id` and see if that helps? If it does, we can narrow the issue down to `listener_id`.

**Comment by janczarn** [comment]
When I use `loadbalancer_id` instead of `listener_id`:

```
resource "openstack_lb_loadbalancer_v2" "lb" {
  name = "lb"
  vip_subnet_id = "${openstack_networking_subnet_v2.subnet.id}"
}

resource "openstack_lb_listener_v2" "listener" {
  name = "listener"
  protocol        = "HTTP"
  protocol_port   = 80
  loadbalancer_id = "${openstack_lb_loadbalancer_v2.lb.id}"
}

resource "openstack_lb_pool_v2" "pool" {
  name = "pool"
  protocol    = "HTTP"
  lb_method   = "ROUND_ROBIN"
  loadbalancer_id = "${openstack_lb_loadbalancer_v2.lb.id}"
}
```

I get this error:

```
1 error(s) occurred:

* openstack_lb_pool_v2.pool: 1 error(s) occurred:

* openstack_lb_pool_v2.pool: Error creating pool: Invalid request due to incorrect syntax or missing required parameters.
```

The pool isn't created.

**Comment by jtopjian** [comment]
@janczarn I recommend running with debug logging enabled. That should provide a better indication of the failure:

```shell
$ TF_LOG=DEBUG OS_DEBUG=1 terraform apply
```

If you'd like, you can post the output to a github gist (after removing any sensitive information) and we can take a look.

**Comment by jtopjian** [comment]
@janczarn Just a quick ping on this one. Are you still seeing the problem?

**Comment by jtopjian** [comment]
@janczarn Github automatically closed this from the merge of #97. If that patch doesn't solve your problem, let us know and we'll re-open this.
